client/app/services/vms.service.js

```diff
@@ -46,6 +46,8 @@ export function VmsService(CollectionsApi, RBAC) {
         'compliances',
         'cpu_affinity',
         'created_on',
+        'custom_actions',
```

Comment:
Yeah, they will.  I have an update for mock api that I will submit that should fix these